src/main/java/Task.java

```diff
@@ -0,0 +1,50 @@
+public class Task {
+    /***
+     * The description of the task.
+     */
+    protected String description;
+    /***
+     * The completion status of the task.
+     */
+    protected boolean isDone;
+
+    /***
+     * Functions in the Task class
+     * @param description Name of the task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /***
+     * Gets the current status of the task within the list.
+     * @return isDone.
```

Comment:
Good effort in starting on commenting your code. Perhaps be more specific in what this method returns, rather than just the name of the variable being returned.